scripts/tests/test_analysis.py

```diff
@@ -12,34 +12,34 @@
 test_sqlite_path = os.path.join(dir, 'test.sqlite')
 
 
-def get_sqlite():
+def sqlite_file_retrieve():
```

Comment:
I don't think this name change is an improvement. "get_sqlite" was more clear. If you prefer, "sqlite_file" would be fine or "get_sqlite_file" would also be fine. The reality is that you're getting the cursor. "connect_sqlite" or "sqlite_cursor" might be more specific.